rover/source/docs/rover-failsafes.rst

```diff
@@ -69,11 +69,38 @@ Advanced Battery Failsafe Settings
 GCS Failsafe (aka Telemetry Failsafe)
 -------------------------------------
 
+Prior Rover-4.4
++++++++++++++++
+
 This failsafe is triggered if the vehicle stops receiving `heartbeat messages <https://mavlink.io/en/messages/common.html#HEARTBEAT>`__ from the ground station for at least :ref:`FS_TIMEOUT <FS_TIMEOUT>` seconds.
 
-- set :ref:`FS_GCS_ENABLE <FS_GCS_ENABLE>` to "1" to enable this failsafe
```

Comment:
dont remove these if you keep the prior to 4.4 section (which we should)

Reply:
That is just moved not removed.
It probably make it more readable if I add the 4.4 specific things at the end. Will update